README.md

```diff
@@ -1,2 +1,37 @@
 # pico-odoo
 Odoo manufacturing add-ons to connect to Pico MES
+
+# Installing the Connector
+1. The connector has a dependency on the OCA queue_job connector.  Ensure that is already installed.
```

Comment:
Can we spell out more specifically what this entails? Doing a quick google search lead me to believe this is another manual install. Saw this: https://github.com/OCA/queue/tree/13.0/queue_job#installation. Or: https://apps.odoo.com/apps/modules/13.0/queue_job/
Or does this show up by default in the apps list if you clear out the Apps filter and search for "Job Queue"? I see that on our Odoo instance now, but guessing that's because it was already installed.